packages/next/src/views/CreateFirstUser/index.client.tsx

```diff
@@ -56,7 +58,14 @@ export const CreateFirstUserClient: React.FC<{
       redirect={admin}
       validationOperation="create"
     >
-      <EmailField autoComplete="email" label={t('general:email')} name="email" required />
+      {loginType === 'emailOrUsername' ? (
+        <>
+          <LoginField type="username" />
+          <LoginField type="email" />
+        </>
+      ) : (
+        <LoginField type={loginType} />
+      )}
```

Comment:
LoginField should likely take required as a prop instead of always being required. This way we can thread props through to CreateFirstUserClient component and then into LoginField. If email is not required, then the CreateFirstUser view should not require it.